canvas_modules/common-canvas/src/common-properties/panels/summary/summary.scss

```diff
@@ -30,11 +31,6 @@
 .properties-summary-link-container {
 	display: flex;
 	align-items: center;
-
-	.properties-icon-button.properties-summary-link-button {
-		padding-left: 0;
```

Comment:
Should the link text be left aligned with the other controls?
<img width="298" alt="image" src="https://user-images.githubusercontent.com/1628100/90423140-cf791780-e070-11ea-8ad6-8c11527c9c43.png">